circle.yml

```diff
@@ -50,6 +52,15 @@ dependencies:
     # Clean up any old images and save the new one
     - I="image-$(date +%j).tgz"; mkdir -p ~/docker; rm ~/docker/*; docker save app:build | gzip -c > ~/docker/$I; ls -l ~/docker
 
+    # build addon
```

Comment:
Thanks les, I'll update

On Mon, Mar 7, 2016 at 1:05 PM, Les Orchard <notifications@github.com>
wrote:

> In circle.yml
> <https://github.com/mozilla/testpilot/pull/451#discussion_r55271822>:
>
> > @@ -50,6 +52,15 @@ dependencies:
> >      # Clean up any old images and save the new one
> >      - I="image-$(date +%j).tgz"; mkdir -p ~/docker; rm ~/docker/*; docker save app:build | gzip -c > ~/docker/$I; ls -l ~/docker
> >
> > +    # build addon
>
> Actually, strike that... I think it needs to go even before line 43
> <https://github.com/meandavejustice/testpilot/blob/447-circleci-addon/circle.yml#L43>,
> before the static asset build - because gulp copies it from static-src to
> static, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/testpilot/pull/451/files#r55271822>.
>